$NetBSD: patch-configure,v 1.19 2024/05/10 15:07:21 taca Exp $

* Do not include "PKG_CONFIG*" in CONFIGURE_OPTIONS.
* Don't autodetect maintainer-zts.
* Shell portability.

--- configure.orig	2024-05-07 14:19:14.000000000 +0000
+++ configure
@@ -4366,6 +4366,10 @@ EOF
    else
     break
    fi
+   case "$CURRENT_ARG" in
+       \'PKG_CONFIG\=*)	CURRENT_ARG="'PKG_CONFIG=@TOOLS_PATH.pkg-config@'";;
+       \'PKG_CONFIG_LIBDIR\=*)	CURRENT_ARG="'PKG_CONFIG_LIBDIR=@PHP_PKGCONFIG_PATH@'";;
+   esac
    printf "%s\n" "$CURRENT_ARG \\" >>config.nice
    CONFIGURE_OPTIONS="$CONFIGURE_OPTIONS $CURRENT_ARG"
   done
@@ -7806,30 +7810,6 @@ EOF
     ;;
   esac
 
-  if test "$APACHE_VERSION" -lt 2004001; then
-    APXS_MPM=`$APXS -q MPM_NAME`
-    if test "$APXS_MPM" != "prefork" && test "$APXS_MPM" != "peruser" && test "$APXS_MPM" != "itk"; then
-
-  enable_zts=yes
-  if test "$pthreads_working" != "yes"; then
-    as_fn_error $? "ZTS currently requires working POSIX threads. We were unable to verify that your system supports Pthreads." "$LINENO" 5
-  fi
-
-    fi
-  else
-    APACHE_THREADED_MPM=`$APXS_HTTPD -V 2>/dev/null | grep 'threaded:.*yes'`
-    if test -n "$APACHE_THREADED_MPM"; then
-
-  enable_zts=yes
-  if test "$pthreads_working" != "yes"; then
-    as_fn_error $? "ZTS currently requires working POSIX threads. We were unable to verify that your system supports Pthreads." "$LINENO" 5
-  fi
-
-    fi
-  fi
-  { printf "%s\n" "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-printf "%s\n" "yes" >&6; }
-
   PHP_VAR_SUBST="$PHP_VAR_SUBST APXS"
 
 else
@@ -84114,7 +84094,7 @@ printf "%s\n" "#define HAVE_TIDYBUFFIO_H
   fi
 
   TIDY_LIBDIR=$TIDY_DIR/$PHP_LIBDIR
-  if test "$TIDY_LIB_NAME" == 'tidyp'; then
+  if test "$TIDY_LIB_NAME" = 'tidyp'; then
 
 printf "%s\n" "#define HAVE_TIDYP_H 1" >>confdefs.h
 
